src/components/modules/blockEvents.ts

```diff
@@ -112,10 +112,8 @@ export default class BlockEvents extends Module {
     if (SelectionUtils.almostAllSelected(block.pluginsContent.textContent)) {
```

Comment:
does this code still required here?

Reply:
I don't think so - do we still need to handle `BlockSettings` being closed if `almostAllSelected` evaluates to true on a `keyup`?